python_scripts/api_requests.py

```diff
@@ -47,20 +68,44 @@ def get_payload(session, url, parameters):
     return resp.json()
 
 
-def scrape_days_from_api(start_date, end_date):
+def scrape_days_from_api(start_date, end_date, db_version, env, skip_write_s3 = True):
+    
+    """ 
+        Scrapes the matrix API for a given period
+        Writes outputs to s3 path as specified by 'env'
+        
+        Parameters:
+            start_date (str): Start date in format %Y-%m-%d
+            end_date (str): End date in format %Y-%m-%d
+                can also be 'eod' to denote end of day
+            env (str): Denotes whether to save results in 
+                production (prod) or development (dev)
+            write_to_s3 (bool): Allow user to skip writing to s3. 
+                Default True
+    """
 
     url = "https://app.matrixbooking.com/api/v1/booking"
     page_size = 2500
     status = ["CONFIRMED", "TENTATIVE", "CANCELLED"]
 
-    params = make_booking_params(
-        start_date, end_date, pageNum=0, pageSize=page_size, status=status
-    )
-
     bookings = []
 
+    # Authenticate session with API
     ses = requests.session()
     matrix_authenticate(ses)
+    
+    # Get booking categories available
+    df_booking_categories = get_booking_categories(ses)
```

Comment:
First make a call to API to get all booking types - to then pass to `make_booking_params`